tests/scale_test/1k-bundle-query.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
I think it would be even better to separate this query from the `tests/scale_test/` to some place like `tests/scale_test/data/2018-04-06`, so that we could decouple the scripts from the data, including the query files and the results. What do you think?